src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -650,12 +650,13 @@ export default class Aircraft {
         let response_end = '';
         const deferred = [];
 
-        for (let i = 0; i < commands.length; i += 1) {
+        for (let i = 0; i < commands.length; i++) {
             const command = commands[i][0];
             const args = commands[i].splice(1);
 
             if (command === FLIGHT_MODES.TAKEOFF) {
                 deferred.push([command, args]);
+                continue
```

Comment:
Good catch!  That is exactly why I had you review these, there's usually something small that gets missed that another set of eyes will see.

You are correct, babel takes care of that at transpile time, but I'd like for that semi colon to be there.